Interpreting-GitHubActions-Results.md

```diff
@@ -0,0 +1,17 @@
+# Interpreting GitHub Actions Results
+
+## Detailed Explanation
+
+Our workflow tests run in parallel as to alleviate the overall pressure on the queue.
+
+  ![GHCI Status](images/ghciSample.png)
+
+In the example above, the lint checks and Mypy checks have failed. However, the rest of the tests have passed.
+
+We also have two checks that have been skipped. These are checks that we have configured to be skipped during the execution of the workflow because they may not be necessary to that particular Pull Request.
```

Comment:
These have been skipped since one of e2e test have failed. If one if e2e tests fails then the remaining e2e tests that are queued or running will be terminated and they will displayed as "skipped"

Reply:
I have changed the explanation.